src/components/BrightcovePlayer.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react';
+import './BrightcovePlayer.css';
+import {connect} from "react-redux";
+import Player from '@brightcove/react-player-loader';
+
+class BrightcovePlayer extends Component {
+
+    success = ({ ref }) => {
+        this.playerRef = ref;
+        if (this.props.selectedVideo !== null) {
+            this.playerRef.catalog.load({
+                sources: [this.props.selectedVideo]
+            });
+        }
+    };
+    shouldComponentUpdate(nextProps) {
+        return this.props.selectedVideo !== nextProps.selectedVideo;
+    }
+    componentDidUpdate(prevProps) {
+        this.playerRef.catalog.getVideo(this.props.selectedVideo, (error, video) => {
+            this.playerRef.catalog.load(video);
+        });
+    }
+    render() {
+        return (
+            <Player attrs={{id: 'videoPlayer'}}
+                    accountId='6027103981001'
+                    playerId='default'
+                    onSuccess={this.success}
```

Comment:
it seems like we could include `videoId: selectedVideo` here and not have to call the catalog method above nor need the code in `componentDidUpdate`?

Reply:
That seems true, though, not sure how that would affect video switching. Probably won't want to re-render the component when switching videos.

Reply:
🤔 yeah I guess the re-rendering might be wasteful